scene/3d/node_3d.cpp

```diff
@@ -230,7 +230,10 @@ void Node3D::_notification(int p_what) {
 
 #ifdef TOOLS_ENABLED
 			if (is_part_of_edited_scene()) {
```

Comment:
Can be single condition.
```suggestion
			if (is_part_of_edited_scene() && !data.gizmos_requested) {
```